Sources/NetworkProtection/KeyManagement/NetworkProtectionTokenStore.swift

```diff
@@ -58,10 +58,6 @@ public final class NetworkProtectionKeychainTokenStore: NetworkProtectionTokenSt
     public func store(_ token: String) throws {
         let data = token.data(using: .utf8)!
         do {
-            try (try? keychainStore.deleteAll()) ?? {
-                // sometimes it fails from the first try: retry once
-                try keychainStore.deleteAll()
-            }()
```

Comment:
We no longer need to delete values before writing them, as the `writeData` function now updates keychain entries instead of failing to save them due to the duplicate item error.